en-GB/docs/find.ndb

```diff
@@ -376,6 +376,8 @@ stfa~Calculates a short-time Fourier analysis of the passed data set~kurzzeitfou
 str~Defines the local string variables in a procedure~procedure local strings~help str
 STR.at(POS)~Return the character at the position POS in the string~methode strings zeichenketten character at()~.at(
 STR.fnd(STR,POS)~Returns the position of the first occurrence of STR in the string, with the search starting from the position POS~methode strings zeichenketten find fnd()~.fnd(
+STR.first~Return the first character of a string first()~.first(
+STR.last~Return the last character of a string last()~.last(

```

Comment:
Suggestion
```
STR.first~Returns the first character of a string~firstch() lastch() begin end last erstes letztes zeichen front back~.first(
STR.last~Returns the last character of a string~firstch() lastch() begin end last erstes letztes zeichen front back~.last(
```

Lacks also the documentation of `firstch()` and `lastch()`. The suggestion also adds a set of keywords as discussed previously